jobs/config.json

```diff
@@ -10262,9 +10262,9 @@
       "--kubernetes-anywhere-kubernetes-version=latest-1.7",
       "--kubernetes-anywhere-upgrade-method=upgrade",
       "--provider=kubernetes-anywhere",
-      "--test_args=--ginkgo.focus=\\[Conformance\\] --minStartupPods=8",
+      "--test_args=--ginkgo.focus=\\[Conformance\\]|\\[Feature:BootstrapTokens\\]|\\[Feature:NodeAuthorizer\\] --minStartupPods=8",
```

Comment:
These didn't exist in v1.7

Reply:
To clarify, are you saying on a cluster with 1.7 nodes and 1.8 control plane, these features would not exist to be tested?

Reply:
I think maybe the concern here is that the test is for upgrading from 1.7 to 1.8. Does the `ginkgo.focus` in `--test_args` start running these tests right after the Kubernetes cluster is brought up, or during the upgrade, or strictly after the upgrade is complete? If it's anything other than the third (when the full upgrade is completely done), then we might be testing 1.8 features with a partially-1.7 control plane.

Reply:
Strictly after upgrade is complete.